### PR TITLE
chore: update electronjs configuration

### PR DESCRIPTION
**Pull request by molant**
# Pull request motivation(s)

The main electronjs.org site is migrating to Docusuarus. This PR updates the
configuration to match more closely the beta-electronjs index one so it takes
advantage of the sitemap and new markup.

### CODE REVIEW

**Review by erickzhao — APPROVED** [review]

File: configs/electronjs.json [hunk 1]
```diff
@@ -1,87 +1,47 @@
...
-      "selectors_key": "docs"
-    },
-    {
-      "url": "https://www.electronjs.org/blog/",
-      "tags": [
-        "blog"
-      ],
-      "selectors_key": "blog"
-    }
+    "https://www.electronjs.org/docs/latest/"
```
Comment (shortcuts) [inline configs/electronjs.json:4]: Is there a reason to only target the latest? Are the old versions available?

(Sorry I can't check myself, I'm on mobile)

Comment (erickzhao) [reply configs/electronjs.json:4]: We're migrating our docs over to Docusaurus. `docs/latest/` is the path for Docusaurus and `/docs/` is currently the path for the old docs. Neither are versioned as of yet. :)

Comment (shortcuts) [reply configs/electronjs.json:4]: Hey @erickzhao 

Thanks, that's what I thought!

If you'd like to keep the old version searchable, it's possible:
- Use the `tags` to make groups that will be used as `facetFilters` in the frontend, see: https://docsearch.algolia.com/docs/config-file/
- Couple that with specific `selector_keys` to target the right DOM elements, see: https://docsearch.algolia.com/docs/config-file/

Comment (molant) [reply configs/electronjs.json:4]: Hi @shortcuts!

Not really looking at making the old version searcheable for now. We want to add docs versioning but we are not there yet so indexing `/latest` is what we want 😊

**Comment by molant** [comment]
Thanks @shortcuts!

**Pull request merged**